src/DotnetWebApiBench.DataAccess/NorthwindDatabaseContext.cs

```diff
@@ -21,24 +21,36 @@
 */
 
 using DotnetWebApiBench.DataAccess.Entity;
+using DotnetWebApiBench.DataAccess.Enums;
 using Microsoft.EntityFrameworkCore;
 using System;
+using System.Diagnostics;
 
 namespace DotnetWebApiBench.DataAccess
 {
     public partial class NorthwindDatabaseContext : DbContext
     {
+        private DbTypeEnum dbtype;
         public NorthwindDatabaseContext()
         {
             this.Database.SetCommandTimeout(TimeSpan.FromMinutes(2));
             this.Database.ExecuteSqlRaw("PRAGMA journal_mode=WAL");
         }
 
-        public NorthwindDatabaseContext(DbContextOptions<NorthwindDatabaseContext> options)
+        public NorthwindDatabaseContext(DbContextOptions<NorthwindDatabaseContext> options, DbTypeEnum dbtype)
             : base(options)
         {
+            this.dbtype = dbtype;
             this.Database.SetCommandTimeout(TimeSpan.FromMinutes(2));
-            this.Database.ExecuteSqlRaw("PRAGMA journal_mode=WAL");
+            if (dbtype == DbTypeEnum.SQLServer)
```

Comment:
For better readability please change the condition to check for SQLite instead of SQLServer - when doing so you can remove the else statement and there would be no empty code.

Reply:
changes applied